frontend/src/pages/workspace/member/Index.tsx

```diff
@@ -50,7 +51,7 @@ function MemberIndex() {
 					<Typography variant="h5" fontWeight="bold">
 						{workspace?.title}{" "}
 						<Typography component="span" variant="inherit" color="primary">
-							{workspaceUserPageList?.pages[0].workspaceUsers.length}
+							{userList.length}
```

Comment:
As `userList` is paginated, we can't determine that the length is same with total users.
It seems it needs to modify backend API to know total number of users.

For reference, you can look at `FindWorkspaceDocumentsResponse` ([Code Pointer](https://github.com/yorkie-team/codepair/blob/main/backend/src/workspace-documents/types/find-workspace-documents-response.type.ts)).